source/prerequisites/sql-server.txt

```diff
@@ -10,4 +10,117 @@ SQL Server Migration Prerequisites
    :depth: 1
    :class: singlecol
 
-SQL Server.
\ No newline at end of file
+Connecting Relational Migrator to SQL Server requires Change Data 
+Capture (CDC) mode enabled. When you set up a new connection, 
+Relational Migrator automatically generates the required SQL code for 
+configuring SQL Server. How you configure SQL Server varies 
```

Comment:
I think we can do a better job of explaining this, Something like: "Before you start a migration job, Relational Migrator will check if the database is configured correctly. If it determines the database is not correctly configured, it will automatically generate a script with the required changes. The commands in this file should be reviewed by a DBA and executed on the database server. This topic provides more details on the required configuration steps."